packages/terminal/package.json

```diff
@@ -1,51 +1,52 @@
 {
-  "name": "@theia/terminal",
-  "version": "0.3.19",
-  "description": "Theia - Terminal Extension",
-  "dependencies": {
-    "@theia/core": "^0.3.19",
-    "@theia/filesystem": "^0.3.19",
-    "@theia/process": "^0.3.19",
-    "@theia/workspace": "^0.3.19",
-    "xterm": "3.9.2"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
-  "theiaExtensions": [
-    {
-      "frontend": "lib/browser/terminal-frontend-module",
-      "backend": "lib/node/terminal-backend-module"
+    "name": "@theia/terminal",
```

Comment:
is `editorconfig` broken? 
it's set to `indent_size = 2`, but if you tamper a json file, it's applying 4 spaces :-(

Reply:
Seems like that. It is even set to 2 twice (once in editorconfig, once in theia settings)